packages/block-editor/src/hooks/layout.js

```diff
@@ -313,23 +313,28 @@ export default {
 	},
 };
 
-function LayoutTypeSwitcher( { type, onChange } ) {
+function LayoutTypeSwitcher( { key, onChange } ) {
 	return (
-		<ButtonGroup>
+		<ToggleGroupControl
+			__next40pxDefaultSize
+			isBlock
+			label={ __( 'Layout Type Select' ) }
```

Comment:
I think a simpler label would be more in line with all the other labels, what do you think?

```suggestion
			label={ __( 'Layout type' ) }
```

Reply:
This is much cleaner and to the point. I made this change along with those you mentioned above.